src/lib/server/clave.ts

```diff
@@ -0,0 +1,28 @@
+export const getUserBuidlBalance = async (userId: string) => {
+  return 0;
+};
+
+export const getClaveInviteLink = async (
+  userEmail: string,
+  userClaveInviteCode: string
+) => {
+  const jwt = signJwt(userEmail);
+
+  return `getclave.io/link/summon?signature=${jwt}&waitlist=${userClaveInviteCode}`;
+};
+
+import jwt from "jsonwebtoken";
+import dayjs from "dayjs";
```

Comment:
shall we move import to top of file or nah

Reply:
great catch!